### ISSUE TITLE
Images for Events and News

### ISSUE THREAD

**Issue by kentonh** [StartupWichita/startupwichita.com#86]
Add ability to upload images to News and Events
Add Images to Index and Show views

Make ```Article URL``` attribute first on the form

News and Events should pull Social Graph data from URL
Pull image, summary, and title information

Populate the rest of the form with Social Graph Information, but allow edits by the user

**Comment by lafentres** [comment]
@kentonh, I'd like to try this one as well.

**Comment by kentonh** [comment]
@lafentres have at it and thanks for the help! With as much as you've contributed to this app, you should register for the [Wichita Hacktoberfest](https://devict-hacktoberfest.herokuapp.com). Might as well get a free shirt and stickers out of the deal.

**Comment by kentonh** [comment]
whoops. reopened

**Comment by lafentres** [comment]
@kentonh, I believe I addressed everything in the pull request I just submitted except for the last part. Can you explain what you mean by "Populate the rest of the form with Social Graph Information, but allow edits by the user" ?

I am pulling everything from information on the event or news item currently. If the event/news item has an image, I use that for the social sharing tags, otherwise I just use the logo. I use the title and content to populate the title and description social sharing tags. 

(I'm definitely checking out the Wichita Hacktoberfest as well. Thanks for the tip!)

**Comment by kentonh** [comment]
What I meant with the Social Graph population was to pre-fill the forms with the information provided by the metadata from Social Graph. Does that make more sense?

**Comment by lafentres** [comment]
I think so. If I am understanding correctly, you'd like part of the form for creating/editing an event or news item to include fields to edit the social sharing info (which would be the social sharing title and description [unless you'd also like the social sharing image to potentially be different from the image associated with the event/news item?]). And those fields should be pre-filled with the defaults. Is that correct?

**Comment by kentonh** [comment]
I think I've done a bad job at describing what I'm going for. Soooo. I'll start over! Forget everything I've said up to this point.

* [This link](http://www.startlandnews.com/2017/10/techstars-spotlight-grit-virtual-builds-3-d-tech-into-2-d-construction-mindset/?utm_source=startupwichita.com) was posted as News.
* Startland includes metadata in the Social Graph format.

What I'm proposing is:

1. When a link is posted, the app would acquire the metadata associated with the link (if there is any)
2. The app would pre-fill the submission form with the data. (Image, Title, Description)
3. At that point a User could simply save the News/Event *or* make changes/additions to the data collected.

Basically, this would let lazy people post on easy mode by simply submitting a link, and the app takes care of the rest.

**Comment by lafentres** [comment]
I get what you mean now. That makes total sense. I'll start working on this.
